index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+	<meta charset="UTF-8">
+	<title>Titlte</title>
+	<link rel="stylesheet" href="style.css">
+</head>
+<body>
+	<div>
+		<header>
+			<h1 align="center">Топ интересных игр</h1>
+		</header>
+	</div>
+		<main class="marg_T">
+			<section>
+				<div class ="container">
+					<p>					
+						<p align = center>Minecraft<br>

```

Comment:
```suggestion
						<p>Minecraft<br>

```

`align = center` - это устаревший атрибут